.github/workflows/security-hardened.yml

```
@@ -50,14 +50,14 @@ jobs:
 
       - name: Upload Hadolint results
-        uses: github/codeql-action/upload-sarif@v3
+        uses: github/codeql-action/upload-sarif@v2
         if: always()
         with:
           sarif_file: hadolint-results.sarif
 
   # Security Auditor: Comprehensive Security Scanning
   security-audit:
-    name: 🔒 Security Auditor
+    name: 🔒 Security Auditor  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)